eos/models/wet.cc

```diff
@@ -366,56 +366,56 @@ namespace eos
 
     /* b->c Wilson coefficients */
     WilsonScanComponent<components::WET::CBLNu>::WilsonScanComponent(const Parameters & p, const Options &, ParameterUser & u) :
-    _e_re_csl(p["cbenue::Re{cSL}"], u),
```

Comment:
Can you please move this change to a separate commit, since it is unrelated to this one?